src/MongoDB/Driver/Manager.cpp

```diff
@@ -466,12 +466,22 @@ const StaticString
 Array HHVM_METHOD(MongoDBDriverManager, __debugInfo)
 {
 	MongoDBDriverManagerData* data = Native::data<MongoDBDriverManagerData>(this_);
+	size_t                        i;
+	mongoc_server_description_t **sds;
+	size_t                        n;
 	Array retval = Array::Create();
 	Array servers = Array::Create();
 
 	retval.add(s_MongoDBDriverManager_uri, (char*) mongoc_uri_get_string(data->m_client->uri));
 
-	mongoc_set_for_each(data->m_client->topology->description.servers, mongodb_driver_add_server_debug_wrapper, &servers);
+	sds = mongoc_client_get_server_descriptions(data->m_client, &n);
+	for (i = 0; i < n; i++) {
+		if (sds[i]->type == MONGOC_SERVER_UNKNOWN) {
+			continue;
```

Comment:
I assume that `mongodb_driver_add_server_debug_wrapper()` never excluded servers in the unknown state, so this is a behavioral change. That said, it's probably quite insignificant.